wire-protoc-tests/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_ARTIFACT_ID=wire-protoc-tests
```

Comment:
Probably doesn't make sense to publish this module so this file is not needed

Reply:
We publish all the other tests module `wire-grpc-tests` and `wire-tests`. I could remove them all in follow up

Reply:
removed it from this one anyway